src/chrome/content/rules/GoNewSoft.com.xml

```diff
@@ -1,7 +0,0 @@
-<ruleset name="GoNewSoft.com">
-        <target host="gonewsoft.com" />
-        <target host="www.gonewsoft.com" />
```

Comment:
Edit [this](https://github.com/MojtabaFaal/https-everywhere/blob/master/src/chrome/content/rules/GoNewCoupon.com.xml) file and add `<target host="gonewsoft.com" />` and `<target host="gonewsoft.com" />`.